partials/anunt.hbs

```diff
@@ -0,0 +1,75 @@
+<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet"
```

Comment:
There are some classes that break the OSUT sites so download the boostrap files and using find in file, search for the classes you used throughout this file and copy them here(for example, m-0, p-5; you can find something similar in assets/css/bootstrap... in infotech repo)